package.json

```diff
@@ -73,6 +74,8 @@
     "sinon": "^2.3.1",
     "style-loader": "^0.18.1",
     "url-loader": "^0.5.8",
+    "video.js": "5.20.1",
+    "videojs5-hlsjs-source-handler": "git+https://github.com/streamroot/videojs5-hlsjs-source-handler.git#1a66611",
```

Comment:
Git version is the [latest release](https://github.com/streamroot/videojs5-hlsjs-source-handler/releases/tag/v0.1.2).  Unfortunately it is not available via npm.